package.json

```diff
@@ -69,15 +68,18 @@
     "@ionic-native/status-bar": "^5.26.0",
     "@ionic-native/vibration": "^5.32.1",
     "@ionic-native/web-intent": "^5.28.0",
-    "@ionic/storage": "^2.2.0",
+    "@ionic/angular": "^4.0.0-beta.6",
```

Comment:
Is there a reason for using this version instead of 4.11.10?

Reply:
I think this was the default version used when I generated the new Ionic 4 project. I've updated it now to use `^4.11.0`.

Reply:
Thanks.